Rock/Web/UI/Controls/Grid/Grid.cs

```diff
@@ -1268,212 +1268,10 @@ protected void Actions_ExcelExportClick( object sender, EventArgs e )
             int rowCounter = 4;
             int columnCounter = 1;
 
-            if ( this.DataSourceAsDataTable != null )
-            {
-                DataTable data = this.DataSourceAsDataTable;
-
-                // print headings
-                foreach ( DataColumn column in data.Columns )
-                {
-                    worksheet.Cells[3, columnCounter].Value = column.ColumnName.SplitCase();
-                    columnCounter++;
-                }
-
-                // print data
-                foreach ( DataRow row in data.Rows )
-                {
-                    for ( int i = 0; i < data.Columns.Count; i++ )
-                    {
-                        worksheet.Cells[rowCounter, i + 1].Value = row[i].ToString().ConvertBrToCrLf();
-
-                        // format background color for alternating rows
-                        if ( rowCounter % 2 == 1 )
-                        {
-                            worksheet.Cells[rowCounter, columnCounter].Style.Fill.PatternType = OfficeOpenXml.Style.ExcelFillStyle.Solid;
-                            worksheet.Cells[rowCounter, columnCounter].Style.Fill.BackgroundColor.SetColor( Color.FromArgb( 240, 240, 240 ) );
-                        }
-                    }
-
-                    rowCounter++;
-                }
-            }
+            if ( DataSourceAsDataTable != null )
+                AddExportDataFromDataTable( worksheet, ref rowCounter, ref columnCounter );
```

Comment:
While having separate methods for the DataTable vs DataView is definitely cleaner code, making this change makes it very difficult to determine what specific functionality you've changed since the diff tools then just show big sections being removed, and another big section being added. 

Also, early on in Rock's life, we had quite the debate over whether to use braces on single line if statements and decided that we would always use them ( I lost ) :).